Header header
uint32 ID
bool active
bool all

